totem/blog/mobile_api.py

```diff
@@ -19,10 +19,10 @@ class BlogPostSchema(ModelSchema):
 
     @staticmethod
     def resolve_header_image_url(obj) -> Optional[str]:
-        if obj.header_image and hasattr(obj.header_image, 'url'):
+        if obj.header_image and hasattr(obj.header_image, "url"):
```

Comment:
Yeah, the linter will get you. This codebase uses "Ruff" (https://github.com/astral-sh/ruff) for formatting. VS Code should have an extension to format on save.